twilio_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'sinatra'
+
+get '/' do
+  'Hello Mudder Fudgers! Currently running version ' + Twilio::VERSION + \
+      ' of the twilio-ruby library'
```

Comment:
Style/MultilineOperationIndentation: Use 2 (not 4) spaces for indenting an expression spanning multiple lines.